pkg/kubecli/replicaset.go

```diff
@@ -37,6 +39,31 @@ type rc struct {
 	resource   string
 }
 
+func (v *rc) GetScale(statefulSetName string, options k8smetav1.GetOptions) (result *autov1.Scale, err error) {
+	result = &autov1.Scale{}
+	err = v.restClient.Get().
+		Namespace(v.namespace).
+		Resource(v.resource).
+		Name(statefulSetName).
+		SubResource("scale").
+		Do().
+		Into(result)
+	return
+}
+
+func (v *rc) UpdateScale(statefulSetName string, scale *autov1.Scale) (result *autov1.Scale, err error) {
```

Comment:
run this regex on the file. 's/statefulSetName/replicaSetName'

Reply:
Done.